tests/integration/test_package_managers.py

```diff
@@ -153,6 +153,17 @@
             ),
             id="gomod_go_generate_imported",
         ),
+        pytest.param(
```

Comment:
I am wondering, if it would be better to rename `test_package_managers.py` to `test_gomod_pkg_manager.py` and create separate file to test pip -> test_pip_pkg_manager.py

Reply:
I had the same thought. I went ahead and split the files, let me know what you think.